packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -191,6 +197,7 @@ const ComboboxField = ({
         isInvalid={isInvalid}
         isTouchAccessible={isTouchAccessible}
         defaultValue={defaultValue}
+        maxHeight={maxHeight}
```

Comment:
These additions will need to be passive with existing maxHeight implementations.

```jsx
<ComboboxField selectAttrs={{ maxHeight: 100 }} />
```

Please ensure each Field variant still supports the previous selectAttrs implementation. This may need to use a fallback.

```jsx
maxHeight={maxHeight || selectAttrs.maxHeight}
```

Reply:
will do